bazel/grpc_python.bzl

```diff
@@ -0,0 +1,70 @@
+"""Custom rules for gRPC Python"""
+
+# Modified from @cython//:Tools/rules.bzl
```

Comment:
These first three lines invite the question "why do we have to write our own `pyx_library` anyway?" and... there should be an answer here. I don't see the term "gRPC" appear in the rest of this file - what's gRPC-specific about this code? If nothing, does this file belong in the Cython repository? If so, we can still add it to this repository today but should there also be a TODO to migrate it to the Cython repository?